src/en/components/card/use-case.md

```diff
@@ -0,0 +1,77 @@
+---
+title: card - Components
+layout: "layouts/component-documentation.njk"
+eleventyNavigation:
+  key: cardEN
+  title: Card
+  locale: en
+  parent: componentsEN
+  otherNames: tile, content card
+  description: A box containing structured, actionable content on a single topic.
+  thumbnail: /images/common/components/preview-card.svg
+  alt: This is an image of the component
+  state: published
+translationKey: "card"
+tags: ['cardEN', 'usage']
+permalink: /en/components/card/
+date: "git Last Modified"
+---
+
+Review these common uses for cards to see if they match the problem you’re solving for. 
+
+## Problems cards solve 
+
+Cards organize information on the same subject in a static box shaped like a playing card.   
+
+Use a card to:
+
+- Group small pieces of related information as a single unit.
+- Present bite-sized previews and summaries of information with a means to obtain more details elsewhere. 
+- Support content discovery by creating more visual interest.
+
+Note: For Canada.ca, avoid using cards in place of the doormats specified in the templates for landing page and theme and topic page.
+
+<div>
+  <h2>Component types</h2>
+  <div>
+    <h3 class="mb-400 mt-400">Link card</h3>
+      {% componentPreview "Link card preview" "py-400" %}
+<gcds-card card-title="Title of the article" tag="Tag" href="#" description="Description or supporting text relating to the headline. Longer text will be truncated with ...">
+<div slot="footer">Context info • metadata</div>
+</gcds-card>
+{% endcomponentPreview %}
+    <p>Use the link card to:</p>
+    <ul class="list-disc mb-400">
+      <li>Structure information on the same topic to progressively offer the reader more details</li>
+      <li>Include a link when there’s further information. Adding a link makes the entire card interactive as a large clickable target.</li>
+      <li>Provide a large, clickable target leading to more information.</li>
+    </ul>
+  </div>
+
+<div>
+<h3 class="mb-400 mt-400">Action card</h3>
+  {% componentPreview "Action card preview" "py-400" "" %}
+  <gcds-card card-title="Title of the article" href="#" type="action" tag="tag" description="Description or supporting text relating to the headline. Longer text will be truncated with ..." img-alt="#">
```

Comment:
Could you please update the card title to use `Card title link` instead to be consistent with the anatomy image?